Documentation/Contributing/development-flow.md

```diff
@@ -243,6 +248,28 @@ Common cases that may need tests:
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
 
+## Linting
+
+The CI will run various linters against your changes to check code, manifests, and docs for syntactic correctness.
+The various linters can be run selectively locally as follows.
+
+Run `make lint` to run all linters. This will require the following CLI tools to be installed on the system:
+
+1. [yamllint](https://www.yamllint.com/)
+2. [shellcheck](https://www.shellcheck.net/)
+3. [pylint](https://www.pylint.org/)
+4. [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2)
+5. [go vet](https://pkg.go.dev/cmd/vet) - part of the golang installation.
```

Comment:
This is part of `go` install, and that is a pretty obvious requirement. I think we can remove this to avoid any confusion it might cause people who might think vet is a separate thing they need.

Reply:
done